ci-templates/gitlab/image-builder.yml

```diff
@@ -233,8 +233,14 @@ capella/base:
     - *prepare-capella
     - *docker
     - mv ../capella.tar.gz ./capella/versions/$CAPELLA_VERSION/$BUILD_ARCHITECTURE/capella.tar.gz
-    - mv ../dropins/* ./capella/versions/$CAPELLA_VERSION/dropins/
-    - mv ../patches/* ./capella/versions/$CAPELLA_VERSION/patches/
+    - >
+      [ -n "$(find ../dropins -maxdepth 0 -type f)" ] && \
+        mv ../dropins/* ./capella/versions/$CAPELLA_VERSION/dropins/ || \
+        echo "No files to move in dropins"'
```

Comment:
```
        mv ../dropins/* ./capella/versions/$CAPELLA_VERSION/dropins/ || \
        echo "No files to move in dropins"'
```

This accepts all errors during moving the files. This is not what we want. We only want to skip it if there are no files. 

In addition, I prefer using `if` statements when it is multi-line.